recipes/kosudoku/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "kosudoku" %}
+{% set version = "1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version | replace('-', '_') }}"
+
+source:
+  url: https://github.com/buzbarstow/kosudoku/archive/refs/heads/master.zip
+  sha256: 3e67e015ecf876898c0832bc9b4e9135cfb0d0bf37edc3d9a723f179af876c40
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  host:
```

Comment:
only python is needed in the host section. Maybe not even that, as you are only copying files not not installing anything

Reply:
I have a poorer understanding of host as opposed to run or build. In which cases is this env used? Without python, the build fails.